music/src/core/data.ts

```diff
@@ -361,19 +366,33 @@ export class MelodyConverter extends DataConverter {
     super(args);
     this.minPitch = args.minPitch;
     this.maxPitch = args.maxPitch;
+    this.disallowPolyphony = args.disallowPolyphony;
```

Comment:
should this have a default of true somehow?

Reply:
I'm actually defaulting it to false since I believe that is more in line with the expected behavior. @tambien can maybe speak to this as well?

Reply:
after discussion, changed to `ignorePolyphony` with default value of true